modules/exploits/windows/http/efs_userid_69.rb

```diff
@@ -0,0 +1,110 @@
+##
+## This module requires Metasploit: http://metasploit.com/download
+## Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Exploit::Remote
+  Rank = GreatRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+                      'Name'    => 'Easy File Sharing Web Server - USERID Remote Buffer Overflow',
```

Comment:
Don't use hard tabs.Stick to the Ruby style guide. You might want to use 2 spaces :)